### PR TITLE
make rspec the default rake task so you can just run 'rake' to run specs

### PR DESCRIPTION
**Pull request by nickhammond**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -21,3 +22,9 @@ module Bundler
 end
 
 Bundler::GemHelper.install_tasks
+
+RSpec::Core::RakeTask.new('spec') do |s|
+  s.rspec_opts = "--color"
+end
+
+task :default => :spec
```
Comment (yfeldblum) [inline Rakefile]: Make sure the file ends in a newline, please. That's my convention for projects I work on.

File: Rakefile [hunk 2]
```diff
@@ -21,3 +22,9 @@ module Bundler
 end
 
 Bundler::GemHelper.install_tasks
+
+RSpec::Core::RakeTask.new('spec') do |s|
+  s.rspec_opts = "--color"
```
Comment (yfeldblum) [inline Rakefile]: Color should be your choice. Run something like `echo '--color' >> ~/.rspec` to add color to all your test runs without needing to configure each project separately.

File: README.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ Ensure the gem dependencies are installed:
 
 Run the tests:
 
-    $ [bundle exec] rspec spec
+    $ [bundle exec] rake
```
Comment (yfeldblum) [inline README.md:27]: Offer both. `rake` is a shorthand. `rspec spec` is canonical and ought to be documented.

**Comment by nickhammond** [comment]
Sorry for the delay, that should address all of your feedback.

**Pull request merged**